src/ts/main.ts

```diff
@@ -20,12 +24,12 @@ $(document).ready(() => {
         startTextAnimation(0);
       }, 2000);
```

Comment:
no stop between loops if you set it to `0`.
Looks better from my point of view.

Reply:
Yep, much better! ;)